tpmutil/run.go

```diff
@@ -30,6 +30,17 @@ import (
 // returning a header and a body in separate responses.
 const maxTPMResponse = 4096
 
+// tcti is a TPM Command Transmission Interface which allows custom handling of
+// a raw TPM command. If a ReadWriter satisfies this interface, commands will be
+// executed through it rather than writing directly to the writer.
+//
+// The primary user of this interface is the "mssim" package. This interface is
+// private for now, as it's unclear if external implementations will need to
+// satisfy it.
+type tcti interface {
+	RunRawCommand([]byte) ([]byte, error)
```

Comment:
Why can't we do this via `io.ReadWriter`?

Reply:
Removed. Managed to do this as an io.ReadWriter :)

Reply:
Removed. Managed to do this as an io.ReadWriter :)